html/sub/demos/basic/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Формула воды — sub — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap">
+  <style>
+    body {
+      background-color: #18191c;
+      color: white;
+      font-family: "Roboto Mono", monospace;
```

Comment:
```suggestion
      font-size: 24px;
      font-family: "Roboto", sans-serif;
```

Демо состоит из одной строки, на размере шрифта не экономим :)